server/chat-commands/info.js

```diff
@@ -1273,7 +1273,7 @@ const commands = {
 
 			if (!pokemon) {
 				let testPoke = Dex.getTemplate(arg);
-				if (testPoke.baseStats) {
+				if (testPoke.exists && testPoke.baseStats) {
```

Comment:
```suggestion
				if (testPoke.exists) {
```

Reply:
Oh, it looks like there was already discussion on this. No, the latter check should be removed. You're right that you don't know for sure if there are situations with no `.baseStats`, but I _do_ know for sure that if there are, they're bugs and should crash so we can fix them.

Reply:
Alright, fair.